### PR TITLE
chore(docs): modify documentation following version bump for quarkus 3.15

### PR DESCRIPTION
**Pull request by joaquinfelici**
Related to https://github.com/camunda/camunda-bpm-platform/issues/4580

### CODE REVIEW

File: content/update/minor/721-to-722/_index.md [hunk 1]
```diff
@@ -268,12 +268,12 @@ This behaviour can be disabled by setting the `skipIsolationLevelCheck` flag to
 
 # Quarkus 3.14 Extension Update
 
-The Camunda Quarkus Extension has been updated to use Quarkus `3.14`. This version brings its own features and changes.
+The Camunda Quarkus Extension has been updated to use Quarkus `3.15`. This version brings its own features and changes.
 For a complete list, see the [Quarkus 3.14.2 Release](https://quarkus.io/blog/quarkus-3-14-2-released) blog post.
```
Comment (joaquinfelici) [inline content/update/minor/721-to-722/_index.md:272]: - [ ] Update when we have release post.

File: content/update/minor/721-to-722/_index.md [hunk 2]
```diff
@@ -266,16 +266,16 @@ This behaviour can be disabled by setting the `skipIsolationLevelCheck` flag to
 
 [See here]({{< ref "/reference/deployment-descriptors/tags/process-engine.md#configuration-properties" >}}) for more details about this and other properties.
 
-# Quarkus 3.14 Extension Update
+# Quarkus 3.15 Extension Update
 
-The Camunda Quarkus Extension has been updated to use Quarkus `3.14`. This version brings its own features and changes.
-For a complete list, see the [Quarkus 3.14.2 Release](https://quarkus.io/blog/quarkus-3-14-2-released) blog post.
+The Camunda Quarkus Extension has been updated to use Quarkus `3.15`. This  version brings its own features and changes.
+For a complete list, see the [Quarkus 3.15.0 Release](https://quarkus.io/blog/quarkus-3-15-0-released) blog post.
```
Comment (joaquinfelici) [inline content/update/minor/721-to-722/_index.md:272]: This hasn't been published yet.
- [ ] Wait until blog post is published before merging.

Comment (mboskamp) [reply content/update/minor/721-to-722/_index.md:272]: Quarkus 3.15.1 blogpost [is available](https://quarkus.io/blog/quarkus-3-15-1-released/).

Comment (mboskamp) [reply content/update/minor/721-to-722/_index.md:272]: ❌ The blog post has a different URL than expected.
```suggestion
For a complete list, see the [Quarkus 3.15 LTS Release](https://quarkus.io/blog/quarkus-3-15-1-released/) blog post.
```

Comment (joaquinfelici) [reply content/update/minor/721-to-722/_index.md]: Updated version and URL.

**Review by psavidis — APPROVED** [review]

**Pull request merged**